command/rm.go

```diff
@@ -80,7 +78,8 @@ func (d Delete) Run(ctx context.Context) error {
 		printError(d.fullCommand, d.op, err)
 		return err
 	}
-	if err := srcURLsBucketVal(srcurls); err != nil {
+	if !hasSameBuckets(srcurls) {
+		err := errors.New("more than one bucket detected")
```

Comment:
could we print more user-friendly message? If I try to remove objects from different buckets (using variadic arguments of course), I'll get this message, which looks like a warning instead of a message that explains what is wrong with the command.